.github/dependabot.yml

```diff
@@ -1,18 +1,19 @@
 version: 2
 updates:
-- package-ecosystem: npm
-  directory: "/"
-  schedule:
-    interval: weekly
-  ignore:
-  - dependency-name: electron
-  - dependency-name: spectron
-  - dependency-name: typescript
-  - dependency-name: sass-loader
-    versions: ["11.x"]
+  - package-ecosystem: "npm"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    ignore:
+      - dependency-name: "electron"
+      - dependency-name: "spectron"
+      - dependency-name: "typescript"
+      - dependency-name: "sass-loader"
+        versions: ["11.x"]
+      - dependency-name: "@types/node"
 
-- package-ecosystem: npm
-  directory: "/nuxt/"
-  open-pull-requests-limit: 0
-  schedule:
-    interval: "weekly"
+  - package-ecosystem: npm
+    directory: "/nuxt"
+    open-pull-requests-limit: 0
+    schedule:
+      interval: "weekly"
```

Comment:
I hope it will stop to suggest upgrades for `nuxt`

Reply:
Why would it stop from this change? If you don't want it to udpate nuxt, why not remove this whole chunk?
```suggestion
```

Reply:
You know, I thought about that. It is OK for me, worst case we can tag the last commit with `nuxt` to easily restore it if we will want to.
I will do that in a different PR, let's merge this PR as is.